src/js/providers/default.js

```diff
@@ -94,9 +94,10 @@ const DefaultProvider = {
     },
 
     sendMediaType: function(levels) {
-        var type = levels[0].type;
-        var isAudioFile = (type === 'oga' || type === 'aac' || type === 'mp3' ||
-            type === 'mpeg' || type === 'vorbis');
+        const level = levels[0];
+        const { type, mimeType } = level;
```

Comment:
level is not used. just destructure levels[0]

`const { type, mimeType } = levels[0];`